packages/slate-theme/test/currency.spec.js

```diff
@@ -29,13 +29,23 @@ describe('currency.formatMoney', () => {
     expect(value).toBe('$100');
   });
 
-  test('Formats a number 1000001 to a string of "$10,000"', () => {
+  test('Formats a number 1000001 to a string of "$10.000"', () => {
```

Comment:
Not sure why you changed this? `no_decimals` meaning no cents value (e.g. `.00`), and the comma is now a `.`?

Reply:
This is for `amount_no_decimals_with_comma_separator` money format which I have set in my store

I get `.` as separator by using the `money` filter

![screen shot 2017-10-30 at 3 38 11 pm](https://user-images.githubusercontent.com/5134470/32197039-649ac750-bd88-11e7-8908-0c00c800129f.png)

Docs also are using a dot

{{ amount_no_decimals_with_comma_separator }} | ✓ | 1.135
-- | -- | --

https://help.shopify.com/manual/payments/currency-formatting

![screen shot 2017-10-30 at 3 43 01 pm](https://user-images.githubusercontent.com/5134470/32197254-15c4f0aa-bd89-11e7-9cf5-a32ef9542c66.png)